cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -1283,8 +1449,29 @@ private boolean tryPullTimeSeriesSchemas(Node node, PullSchemaRequest request) {
     List<Node> coordinatedNodes = QueryCoordinator.getINSTANCE().reorderNodes(partitionGroup);
     for (Node node : coordinatedNodes) {
       try {
-        List<PartialPath> paths =
+        GetAllPathsResult result =
             getMatchedPaths(node, partitionGroup.getHeader(), pathsToQuery, withAlias);
+
+        List<PartialPath> partialPaths = new ArrayList<>();
+        if (result != null) {
+          result.pathToPaths.forEach(
+              (k, v) -> {
+                v.forEach(
+                    p -> {
```

Comment:
In fact, when `vector` functionality is added, using partialPath directly here may not work for all paths.
maybe we need a detailed discussion to adapt Vector and CMManager

Reply:
merge the PR first ， and modify it after vectoring is complete  ?  or 
after functionality is added, merge this pr.

Reply:
I am OK with the first choice~